components/cards/CardGridContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import Box from '@mui/material/Box'
+import { ReactNode } from 'react'
+
+type Props = {
+  children: ReactNode
+}
+
+const CardGridContainer = ({ children }: Props) => {
```

Comment:
is this meant to be used somewhere?

Reply:
are you referring to the Props type I made? It's convenient in case I want to add another prop to the component. But I also could've written it like `({ children }: { children: ReactNode })`  and do away with that Props type altogether.

(feel free to clarify if I misunderstood!)